app/src/main/res/values-de/strings.xml

```diff
@@ -230,66 +230,66 @@
     <string name="ma">%1$d mA</string>
     <string name="ma_hint">mA</string>
     <string name="overheat">Überhitzung</string>
-    <string name="overcool">Unterkühlen</string>
+    <string name="overcool">Unterkühlung</string>
     <string name="number_of_cycles_android">Anzahl der Zyklen (Android): %1$d</string>
     <string name="number_of_cycles_android_overlay">Anzahl der Zyklen (Android)</string>
-    <string name="discharge_current_settings_channel">Entladungsstrom</string>
-    <string name="discharge_current_ma">Entladungsstrom: %1$d mA</string>
+    <string name="discharge_current_settings_channel">Entladestrom</string>
+    <string name="discharge_current_ma">Entladestrom: %1$d mA</string>
     <string name="auto_start_boot">Automatischer Start (Boot)</string>
     <string name="auto_start_open_app">Automatischer Start (Anwendung öffnen)</string>
     <string name="auto_start_update_app">Automatischer Start (Anwendung aktualisieren)</string>
-    <string name="start_capacity_info_service">Start Capacity Info Service</string>
-    <string name="stop_capacity_info_service">Stopp Capacity Info Service</string>
-    <string name="restart_capacity_info_service">Neustart Capacity Info Service</string>
-    <string name="stop_overlay_service">Stopp Overlay Service</string>
-    <string name="restart_overlay_service">Neustart Overlay Service</string>
-    <string name="history">Geschichte</string>
-    <string name="add_history">Hinzufügen Geschichte</string>
-    <string name="add_ten_history">Hinzufügen 10 Geschichte</string>
-    <string name="add_fifty_history">Hinzufügen 50 Geschichte</string>
-    <string name="clear_history">Geschichte löschen</string>
-    <string name="history_cleared_successfully">Geschichte erfolgreich bereinigt!</string>
-    <string name="error_clearing_history">Fehler beim Löschen der Historie!</string>
-    <string name="clear_the_history_dialog_message">Historie löschen?</string>
-    <string name="export_history">Geschichte exportieren</string>
+    <string name="start_capacity_info_service">Capacity Info Service starten</string>
+    <string name="stop_capacity_info_service">Capacity Info Service stoppen</string>
+    <string name="restart_capacity_info_service">Capacity Info Service neustarten</string>
+    <string name="stop_overlay_service">Overlay Service stoppen</string>
+    <string name="restart_overlay_service">Overlay Service neustarten</string>
+    <string name="history">Verlauf</string>
+    <string name="add_history">Verlauf hinzufügen</string>
+    <string name="add_ten_history">10 Verlauf hinzufügen</string>
```

Comment:
What is meant here? I didn't find that in the app.

Reply:
This is in Debug)